source/_docs/automation/trigger.markdown

```diff
@@ -279,6 +279,22 @@ The `for` template(s) will be evaluated when the `value_template` becomes `true`
 Rendering templates with time (`now()`) is dangerous as trigger templates only update based on entity state changes.
 </div>
 
+
+As an alternative, providing you include the sensor [time](/integrations/time_date/) in your configuration, you can use the following template:
+
+{% raw %}
+
+```yaml
+automation:
+  trigger:
+    platform: template
+    value_template: "{{ (as_timestamp(states.sensor.time.last_changed) - as_timestamp(states.YOUR.ENTITY.last_changed)) > 300 }}"
```

Comment:
We should not use `states.sensor.time.last_changed` as per https://www.home-assistant.io/docs/configuration/templating/#states

Reply:
Interestingly I never use the form `states.sensor.time.last_changed` in my code but that is how it was suggested in the forum so I bowed to what I thought was their better knowledge.

Reply:
Err... DO I need to do something to 'accept' your suggested change?
(I'm still learning my way around Github...)

Reply:
Sorry, you are right, not sure where my mind was when I wrote that comment. 🤒